greek/offer_marshmallow_a_cigar/set_it_on_fire/set_it_on_fire.md

```diff
@@ -4,4 +4,8 @@
 
 Και τώρα; Βγάζεις το κεφάλι από το άνοιγμα και αυτό που
 αντικρύζεις είναι τόσο φρικιαστικό που θέλεις να ξαναμπείς μέσα
-και να κλείσεις το άνοιγμα για πάντα.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.